packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -457,26 +474,39 @@ export async function copyProjectTemplateAndReplace(
     }
   }
 
-  // shared assets
-  if (fs.existsSync(path.join(sharedPath, 'assets'))) {
-    await copyAndReplaceAll(
-      path.join(sharedPath, 'assets'),
-      destPath,
-      path.join(windowsDir, newProjectName, 'Assets'),
-      templateVars,
-      options.overwrite,
-    );
-  }
+  if (!options.useWinUI3) {
+    // shared assets
+    if (fs.existsSync(path.join(sharedPath, 'assets'))) {
+      await copyAndReplaceAll(
+        path.join(sharedPath, 'assets'),
+        destPath,
+        path.join(windowsDir, newProjectName, 'Assets'),
+        templateVars,
+        options.overwrite,
+      );
+    }
 
-  // shared src
-  if (fs.existsSync(path.join(sharedPath, 'src'))) {
-    await copyAndReplaceAll(
-      path.join(sharedPath, 'src'),
-      destPath,
-      path.join(windowsDir, newProjectName),
-      templateVars,
-      options.overwrite,
-    );
+    // shared src
+    if (fs.existsSync(path.join(sharedPath, 'src'))) {
+      await copyAndReplaceAll(
+        path.join(sharedPath, 'src'),
+        destPath,
+        path.join(windowsDir, newProjectName),
+        templateVars,
+        options.overwrite,
+      );
+    }
+  } else {
+    //TODO update template to use shared assets and src
+    if (fs.existsSync(path.join(srcPath, 'MyApp'))) {
+      await copyAndReplaceAll(
+        path.join(srcPath, 'MyApp'),
+        destPath,
+        path.join(windowsDir, newProjectName),
+        templateVars,
+        options.overwrite,
+      );
+    }
```

Comment:
This seems worth doing before merging

Reply:
updated to use shared assets, src is different.